app/ModelFunctions/AlbumFunctions.php

```diff
@@ -543,7 +545,9 @@ public function getToplevelAlbums()
 			if ($id == 0) {
 				$return['albums'] = Album::where('owner_id', '=', 0)
 					->where('parent_id', '=', null)
-					->orderBy(Configs::get_value('sorting_Albums_col'), Configs::get_value('sorting_Albums_order'))->get();
+					->orderBy(Configs::get_value('sorting_Albums_col'), Configs::get_value('sorting_Albums_order'))
+					->get()
+					->sortBy(Configs::get_value('sorting_Albums_col'), SORT_NATURAL, (Configs::get_value('sorting_Albums_order') === 'DESC'));
```

Comment:
So, given your follow-up timings from #321, I wonder if we should be doing orderBy here and in other such places at all... I think we'll need to try three versions:

* plain `orderBy` (current master)
* `orderBy` plus `sortBy` (your PR)
* just `sortBy`

I'll see if I can work on that tonight...

Reply:
Removing the OrderBys would simplify the code, I just did not want to get too crazy as an outside contributor.